src/domain/index.ts

```diff
@@ -88,20 +71,62 @@ export type Message =
       parsed: string;
     };
 
+export function getTransactionStatusText(status: Transaction["status"]): string {
```

Comment:
How we display the text in the UI shouldn't be part of the domain. If you need to reuse this, it should be a component: `TransactionStatus`.

Reply:
> How we display the text in the UI shouldn't be part of the domain.

Agreed.

> If you need to reuse this, it should be a component: `TransactionStatus`.

I don't agree with this. I think that having a util function as now is more versatile. With a util function, we can get the text and use it to build a msg for the user, show a log, or anything else, while a component limits the usage to JSX.

Reply:
Done. Both utils moved to a new module `utils/labels.ts`.